.github/workflows/master_build_pr.yml

```diff
@@ -168,249 +103,37 @@ jobs:
-            print(e)
-            post_to_slack(e)
-            exit(1)
-          EOF
-            set +e
-          }
-          installGoDependencies() {
-            go version
-            go install golang.org/x/tools/cmd/goimports@latest
-            which go
-          }
-          infraProvisionBuild() {
-            set -e
-            lintCheck
-            IMAGE="$1"
-            requiredInputs="1"
-            validateInputs "$requiredInputs" "$@" 
-            mkdir -p build/tools
-            wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/6.0.0/openapi-generator-cli-6.0.0.jar -O build/tools/openapi-generator-cli-6.0.0.jar
-            gradlewPythonwheel
-            imageScan $IMAGE
-            set +e
-          }
-          lintCheck() {
-            flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-            flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-          }
-          pythonDockerBuild() {
-            set -e
-            IMAGE="$1"
-            rm pip.conf
-            lintCheck
-            gradlewPythonwheel
-            imageScan $IMAGE
-            set +e
-          }
-          validateInputs() {
-            local required=($1)
-            shift
-            for index in "${required[@]}"; do
-              if [ -z "${!index}" ]; then
-                echo "Error: Required Argument at index $index is null or empty."
-                exit 1
-              fi
-            done
-          }
-          build() {
-            type="$1"
-            check=$(grep "$type" convoy.yaml)
-            if [[ -z "$check" ]]; then
-              return
-            fi
-            version=$(yq '.version' convoy.yaml)
-            yq e ".generates.$type[] | [.name, .buildFunction, .extension, .dockerFile, .filePath, .baseImage] | @csv" convoy.yaml | sed 's/,/ /g' > artifacts.txt
-            lineNumber=1
-            while :; do
-              echo ------------------------------
-              artifactData=$(sed "$lineNumber!d" artifacts.txt)
-              if [[ -z "$artifactData" ]]; then
-                break
-              fi
-              read -r name buildFunction ext file filePath baseImage <<< "$artifactData"
-              echo "Name: $name"
-              echo "buildFunction: $buildFunction"
-              echo "Ext: $ext"
-              echo "Version: $version"
-              echo "dockerFile: $file"
-              echo "filePath: $filePath"
-              echo "baseImage: $baseImage"
-              $buildFunction "$name" "$ext" "$version" "$file" "$filePath" "$baseImage"
-              echo "$type $name done"
-              echo ------------------------------
-              lineNumber=$((lineNumber+1))
-            done
-            rm artifacts.txt
-          }
-          set +e
-          build "artifacts"
-          build "dockerImages"
-          build "helmCharts"
+          bash ./ci-scripts/bash/build.sh Build
```

Comment:
Why `B` in uppercase?

Reply:
Many usecase comes for terraform{Build} and terraform{BuildAndPush} Line 1311 in build.sh